### PR TITLE
Address justify left regression

### PR DESCRIPTION
**Pull request by blackfalcon**
Changes to be committed:
modified:   src/style.scss

# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #53 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

This PR addresses a regression where the `justifyLeft` attribute was no longer working. It appears with a recent update CSS code for the `justifyLeft` attribute was removed? 

Just reason 10748 for using UI regression testing. 

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [x] Bug fix

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/style.scss [hunk 1]
```diff
@@ -73,12 +73,23 @@
...
   width: 100%;
   padding-right: var(--auro-size-md);
 }
 
-:host([justifyRight]) {
+:host([justifyRight]),
+:host([justifyLeft])  {
   .triggerContent {
     padding-right: 0;
   }
```
Comment (jason-capsule42) [inline src/style.scss:95]: I find this code confusing. Looking at the layout rendered it seems this padding is to keep the text label from getting too close to the carrot/chevron.

I am not sure I understand why we want to keep them from being too close when we don't justify left/right but collapse that space when we do?

@leeejune can you comment here? Is this intended?

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: src/style.scss (hunk 1 above)
Comment (leeejune) [reply src/style.scss:95]: The justify version is meant to have different spacing. This is outlined in the blueprint.

Comment (blackfalcon) [reply src/style.scss:95]: There is no legacy design spec that defines this. It was just made up as we created the new versions. 

The scope of this issue is to fix the accidentally deleted references to `justifyLeft`. 

The work @leeejune is doing to redefine the blueprint spec would be applied to a new issue. The plan is to revamp the accordion element entirely.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**